accurest-core/src/test/groovy/io/codearte/accurest/dsl/WiremockGroovyDslSpec.groovy

```diff
@@ -313,6 +315,168 @@ class WiremockGroovyDslSpec extends WiremockSpec {
     ''')
 	}
 
+	def "should generate request with urlPath and queryParameters for client side"() {
+		given:
+			GroovyDsl groovyDsl = GroovyDsl.make {
+				request {
+					method 'GET'
+					urlPath($(client("users"), server("items"))) {
+						queryParameters {
+							parameter 'limit': $(client(equalTo("20")), server(containing("10")))
```

Comment:
I really like the DSL!

We could do one check though, like we are doing in the main part of Accurest. That if you provide `containing`, `matching`, `notMatching` etc. then we will check the other side if they actually match.

So if you have

```
$(client(equalTo("20")), server(containing("10"))
```

this should fail at compilation level cause

`20` doesn't contain `10`

WDYT?